### PR TITLE
Fixed Grammar Mistakes in README.md

### PR DESCRIPTION
**Pull request by IsaacLK**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ It is also possible to view all test PDF files on the right side by opening:
 
 ## Building PDF.js
 
-In order to bundle all `src/` files into two production scripts and build the generic
```
Comment (Snuffleupagus) [inline README.md:82]: Sorry, but why change this?

File: README.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ be loaded by `pdf.js`. The PDF.js files are large and should be minified for pro
 ## Using PDF.js in a web application
 
 To use PDF.js in a web application you can choose to use a pre-built version of the library
-or to build it from source. We supply pre-built versions for usage with NPM and Bower under
+or to build it from the source. We supply pre-built versions for usage with NPM and Bower under
```
Comment (Snuffleupagus) [inline README.md:98]: "build from source" seem to be a very commonly used phrase, so I'm not sure that we need to change this!?

**Review by Snuffleupagus — CHANGES_REQUESTED** [review]
Please also see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits, and remember to write a good *commit message*.

File: README.md (hunk 2 above)
Comment (IsaacLK) [reply README.md:98]: It just sounded better in my head :)
Should I remove this?

Comment (IsaacLK) [reply README.md:98]: I'll remove it.

Comment (IsaacLK) [reply README.md:98]: I'm going to close this and make a new PR, just to simplify things.

**Comment by IsaacLK** [comment]
I'm going to close this PR and make a new one to simply things.

**Pull request closed without merging**